### DISCUSSION TITLE
Site failing to load for multiple users

### DISCUSSION THREAD

**Discussion by aplchian** [vercel/community#346] [Help] [answered]
Hello,

This morning we started to get reports that some users were getting various connection errors when requesting our site. Is anyone else experiencing these errors? Any ideas what's going on?

![image](https://user-images.githubusercontent.com/7388748/153288198-ffcccffd-a39d-4e72-92bd-20d338f192bb.png)
![image](https://user-images.githubusercontent.com/7388748/153288317-b88e6711-1ac4-4d09-a68a-2403b798184f.png)
![image](https://user-images.githubusercontent.com/7388748/153288467-fa098c88-5673-4512-a624-bfd4147d1b5e.png)

**Comment by aplchian** [comment]
Also, our bandwidth usage is Way up today.. i dont know if this is related.

**Comment by Pieparker** [comment, marked as answer]
These sorts of errors would indicate errors in resolving DNS records. Check that they're correct!

Hard to tell without knowing what the site is! If issues persist, you may wish to share the project ID.
